scripts/delete_lambda_versions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Delete all published AWS Lambda versions except for "$LATEST" in the current
+deployment.
+"""
+
+import logging
+
+from azul.deployment import (
+    aws,
+)
+from azul.lambdas import (
+    Lambdas,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    lambdas = Lambdas().list_lambdas(deployment_only=True)
+    for lambda_ in lambdas:
```

Comment:
```suggestion
    functions = Lambdas().list_lambdas(deployment_only=True)
    for function in functions:
```

I want to move away from referring to AWS Lambda functions as "lambdas".